### ISSUE TITLE
Tezos faucet not working

### ISSUE THREAD

**Issue by MunimIftikhar** [tacoinfra/tezos-faucet#3]
I've tried to fund the token to my wallet address multiple times but I don't receive any TEZ tokens. Multiple people are facing the same issue. Please help regarding this.

**Comment by drchrispinnock** [comment]
Can you provide more detail please? are you using the browser method and which test network please?
We've tried all four test networks and received funds.

**Comment by MunimIftikhar** [comment]
Yes. I'm using the site: https://faucet.ghostnet.teztnets.com/ and ghostnet testnet network. When I try to fund the tokens to my Temple wallet, I don't receive any tokens even after some time.

**Comment by drchrispinnock** [comment]
I had a similar problem with Temple over the weekend. It cannot read balances from rpc.ghostnet.teztnets.com. It can read balances from rpc.ghostnet.tzboot.net but doesn't seem to be able to transact. I've asked the Temple team to look at this - the last update I had was there was a potential bug.

I would suggest that you go to tzstats.com, switch to ghost net and check your wallet address there. If the balance is still not what you expect on the block explorer, please ping me back here. I will need your public key hash (do not share your secret key).

**Comment by MunimIftikhar** [comment]
The explorer shows me the expected balance. This is the Temple wallet issue. Thank you so much for helping me with this!
